src/screens/ImportSeedPhraseSheet.js

```diff
@@ -166,7 +166,7 @@ export default function ImportSeedPhraseSheet() {
             color={isSecretValid ? colors.appleBlue : colors.dark}
             onChangeText={handleSetSeedPhrase}
             onFocus={handleFocus}
-            onSubmitEditing={handlePressImportButton}
+            onSubmitEditing={() => handlePressImportButton()}
```

Comment:
no

Reply:
guard against `forceColor` being an object in the def of `handlePressImportButton`